web/src/containers/ApdPreviousActivityTableTotal.js

```diff
@@ -0,0 +1,115 @@
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+
+import { formatMoney } from '../util/formats';
+import { TABLE_HEADERS } from '../constants';
+
+const ApdPreviousActivityTableMMIS = ({ totals }) => {
+  const years = Object.keys(totals);
+
+  return (
+    <Fragment>
+      <h3>Grand totals: Federal HIT, HIE, MMIS</h3>
+      <div className="table-frozen-wrapper table-frozen-narrow-header mt3">
```

Comment:
```suggestion
      <div className="table-frozen-wrapper table-frozen-narrow-header">
```
This `mt3` should be removed in order to keep consistent with the table spacing styles above in this section.

Reply:
Oops!  I copied over the previous MMIS table wholesale and forgot to take out the margin.  It was originally meant to space out tables from each other, but I guess we get the right amount of spacing from the header.